wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md

```diff
@@ -35,13 +35,15 @@ This is mostly due to changes to the judgement system. For example, Geki and Kat
 |  |  |
 | :-- | :-: |
 | Breaks backwards compatibility | Yes |
-| Classic mod revert support | No |
+| Classic mod revert support | Yes |
 | Intentionally changed | Yes |
 | Needs further consideration | Yes |
 
-### All game modes use the same grading system
+### All game modes except osu!catch use the same grading system
```

Comment:
would rather just copy directly from the recent news post

```markdown
In osu!(stable), the accuracy (and judgement) requirements for each [grade](/wiki/Gameplay/Grade) are as follows:

| Grade | osu! / osu!taiko | osu!catch | osu!mania |
| :-: | :-- | :-- | :-- |
| SS | 100% | 100% | 100% |
| S | ≥90% (≤1% MEHs/50s, no misses) | ≥98% | ≥95% |
| A | ≥80% (no misses) or ≥90% | ≥94% | ≥90% |
| B | ≥70% (no misses) or ≥80% | ≥90% | ≥80% |
| C | ≥60% | ≥85% | ≥70% |

Meanwhile, osu!(lazer) has these accuracy cutoffs now:

| Grade | osu! / osu!taiko / osu!mania | osu!catch |
| :-: | :-- | :-- |
| SS | 100% | 100% |
| S | ≥95% | ≥98% |
| A | ≥90% | ≥94% |
| B | ≥80% | ≥90% |
| C | ≥70% | ≥85% |
```

![image](https://github.com/ppy/osu-wiki/assets/36758269/bf88adfe-2f6d-425f-acee-cbbcfa92d925)

Reply:
What about this title?

```suggestion
### Grading system is different
```

Reply:
Actually the changes broke something with the pipes, will fix later, I'm done for today

Reply:
Fixed, waiting for feedback on title change

Reply:
ok as is